src/services/token.js

```diff
@@ -11,7 +11,12 @@ class TokenService {
   }
 
   async init () {
-    this.token = await getToken()
+    this.token = await getToken(window.web3.eth.defaultAccount)
+    if (!this.token) {
+      return false
```

Comment:
this check shouldn't be necessary, since init is only called at one place on load